app/web/components/pages/config/financialAccounts/FinancialAccountConfig.js

```diff
@@ -2,7 +2,8 @@
 import React, { useState, useEffect } from 'react';
 import Avatar from '@material-ui/core/Avatar';
 import { Edit, Save, Delete } from '@material-ui/icons';
-import { Select, MenuItem, Checkbox } from '@material-ui/core';
+import { Select, Checkbox } from '@material-ui/core';
+import MenuItem from '@material-ui/core/MenuItem';
```

Comment:
Thats good. If you are improving that, lets make it consistent and do the same for Select and Checkbox